min_range: 58	# min range of radar points (bin)
radar_resolution: 0.0432	# resolution of radar bins in meters per bin

cart_resolution: 0.25	# meters per pixel
cart_pixel_width: 1000	# height and width of cartesian image in pixels
interp: true

# cen2018 feature parameters
zq: 3.0
sigma_gauss: 17

# cen2019 feature parameters
max_points: 10000

keypoint_extraction: 0	# 0: cen2018, 1: cen2019, 2: orb

# ORB feature parameters
patch_size: 21	# width of patch in pixels in cartesian radar image
nndr: 0.65	# Nearest neighbor distance ratio

# RANSAC
threshold: 0.35
inlier_ratio: 0.90
max_iterations: 100

# MDRANSAC
max_gn_iterations: 10
md_threshold: 0.1225

# DOPPLER
beta: -0.0478125
